Resources/config/services.yml

```diff
@@ -1,32 +1,19 @@
 parameters:
-    stadline_status_page.symfony.requirement.class: Stadline\StatusPageBundle\Requirements\SymfonyRequirements
-    stadline_status_page.requirement.collections.class: Stadline\StatusPageBundle\Requirements\RequirementCollections
-    stadline_status_page.version.requirement.class: Stadline\StatusPageBundle\Requirements\VersionRequirements
-    stadline_status_page.api.requirement.class: Stadline\StatusPageBundle\Requirements\ApiRequirements
-    stadline_status_page.api.factory.class: Stadline\StatusPageBundle\ApiStatus\ApiStatusFactory
+    stadline_status_page.symfony.requirement.class: "Stadline\StatusPageBundle\Requirements\SymfonyRequirements"
+    stadline_status_page.requirement.collections.class: "Stadline\StatusPageBundle\Requirements\RequirementCollections"
+    stadline_status_page.version.requirement.class: "Stadline\StatusPageBundle\Requirements\VersionRequirements"
 
 services:
     stadline_status_page.version.requirement:
-        class: %stadline_status_page.version.requirement.class%
+        class: "%stadline_status_page.version.requirement.class%"
```

Comment:
Be careful, with phpunit/phpunit > 5, all YAML files should be simple quoted.

I had some troubles in decathloncoach with it.

see https://github.com/stadline/mygeonaute2/commit/15acf74a95bd9ef5737a7297402a1dc7e7a3f18c